backend/config/config.js

```diff
@@ -10,14 +10,20 @@ var all = {
             useMongoClient: true,
         },
     },
-    sessionSecret: process.env.SESSION_SECRET || 'fghjdfjkdf785a-jreu'
+    sessionSecret: process.env.SESSION_SECRET || 'fghjdfjkdf785a-jreu',
+    sentry: require('./sentry'),
```

Comment:
I think this does not make sense to store objects in the config, it should stay a pure JavaScript object (i.e, serializable with `JSON.stringify`)

Reply:
That's the case.

Reply:
Oh, my bad, you are right 🙂